src/functionalTests/resources/application.properties

```diff
@@ -82,4 +82,4 @@ cos.consentOrder.approved=${case.orchestration.api}/documents/consent-order-appr
 cos.preview.consentOrder.not.approved=${case.orchestration.api}/documents/preview-consent-order-not-approved
 
 #ConsenrOrderNotApproved
-cos.consentOrder.not.approved=${case.orchestration.api}/documents/consent-order-not-approved
+cos.consentOrder.not.approved=${case.orchestration.api}/documents/consent-order-not-approved
```

Comment:
git compares are awesome, theres a type in the comment above these `ConsenrOrderNotApproved`